src/Glpi/Api/HL/Router.php

```diff
@@ -621,7 +621,7 @@ public function handleRequest(Request $request): Response
             }
         }
 
-        if ($CFG_GLPI['enable_hlapi']) {
+        if (Config::isHlApiEnabled()) {
```

Comment:
Might need a use statement for this file

Reply:
Indeed, added in last commit